tests/unit/test_charm_2.py

```diff
@@ -29,6 +66,12 @@ def test_generate_wp_secret_keys(self):
         """
         self.harness.begin()
         secrets = self.harness.charm._generate_wp_secret_keys()
+        self.assertIn(
+            "default_admin_password",
+            secrets,
+            "wordpress should generate a default admin password"
+        )
+        del secrets["default_admin_password"]
```

Comment:
For my understanding, why is this being deleted?

Reply:
Because I wrote the following test statements first, I don't want to rewrite this entire test case when adding tests for default_admin_password :D

Reply:
👍